domain.yml

```diff
@@ -69,6 +71,27 @@ slots:
   travel_flight_class:
     type: any
     influence_conversation: false
+  conversation_rating:
+    type: categorical
+    influence_conversation: false
+    initial_value: 0
+    values:
+    - 1
+    - 2
+    - 3
+    - 4
+  travel_flight_distancecategory:
+    type: categorical
+    influence_conversation: true
+    values:
+    - short
+    - medium
+    - long
+    - extended
+    - unknown
+  travel_flight_km:
+    type: text
+    influence_conversation: true
```

Comment:
Did you mean to add these back in?

Reply:
oh dear - sorry, i merge-failed them back in...

Reply:
No worries, I figured :) Aside from that this looks good to me, I'm assuming you've tried it out and made sure it works as expected